### DISCUSSION TITLE
🏗️ Public Work Session #2 - L2 Research (5 May 2023)

### DISCUSSION THREAD

**Discussion by isaaczaak** [ourzora/nouns-builder#245] [General]
**Figjam:**
- ORs vs ZK
- ORs lacking fraud proofs and require a third-party layer eg LayerZero for instant finality to send messages from L2 to L1 
- Decision-making matrix
  - Security
  - Cost-savings benefit
  - User experience
  - Developer experience

![image](https://user-images.githubusercontent.com/108648359/236377515-61580b1f-6c06-416f-878c-2ebd1531da62.png)

**Recording:**
https://drive.google.com/file/d/1qgi7VQLdFKhcvW-Wyk56KkE_HfQw1gbC/view?usp=sharing

**Comment by Valcoholics** [comment]
Thanks!

**Comment by irlinantowoharsteve** [comment]
interesting

**Comment by wasabu0x** [comment]
good bro

**Comment by Mick551** [comment]
interesting

Thanks

**Comment by dert0n-web** [comment]
Thanks!

**Comment by RajhonAb** [comment]
interesting
